youtube_dl/aes.py

```diff
@@ -80,50 +111,75 @@ def aes_cbc_encrypt(data, key, iv):
         block += [remaining_length] * remaining_length
         mixed_block = xor(block, previous_cipher_block)
 
-        encrypted_block = aes_encrypt(mixed_block, expanded_key)
+        encrypted_block = _aes_encrypt(mixed_block, expanded_key)
         encrypted_data += encrypted_block
 
         previous_cipher_block = encrypted_block
 
     return encrypted_data
 
 
-def key_expansion(data):
+def _aes_gcm_decrypt_and_verify(data, key, tag, nonce):
     """
-    Generate key schedule
+    Decrypt with aes in GBM mode and checks authenticity using tag
```

Comment:
Since #29201 is not merged, youtube-dl does not have any need for gcm

Reply:
True, since I did it I just want it out somewhere...